scripts/source-maintenance.sh

```diff
@@ -216,7 +230,15 @@ git grep "ifn?def .*_SQUID_" |
 #
 # Scan for file-specific actions
 #
-for FILENAME in `git ls-files`; do
+FORKPOINT=`git fork-point`
+if test "x$CHANGEDONLY" = "xyes" && test "x$FORKPOINT" != "x" ; then
+    FILES=`git diff -r $FORKPOINT --name-only`
+else
+    FILES=`git ls-files`
+fi
+echo "files: $FILES"
+exit 0
```

Comment:
Please remove and retest that everything still works.

Reply:
I'm not sure I understand what you mean. Are you objecting to this feature being added?